README.md

```diff
@@ -39,6 +39,10 @@ Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](../../issues/).
 
+[See my landing page here!](https://aron-helu.github.io/landing-page/
+)
+Copyright &copy; by Aaron-A<sup>3</sup>
+
```

Comment:
Please add your link under a `Live Demo` header. Having your link in this section makes it really difficult to find the link to the deployed app, which is the whole goal of this exercise. Also, I would recommend you remove or edit your copyright properly because it looks really strange.